### PR TITLE
doc: adjusted download flow to new release model

### LINKED ISSUE #2394: [Bug] getting-started guide not updated to latest release model
## Describe the bug

[getting-started.md#getting-the-firecracker-binary](https://github.com/firecracker-microvm/firecracker/blob/master/docs/getting-started.md#getting-the-firecracker-binary) is not updated to latest release model

## To Reproduce

Follow the steps provided in the [getting-started.md#getting-the-firecracker-binary](https://github.com/firecracker-microvm/firecracker/blob/master/docs/getting-started.md#getting-the-firecracker-binary)

## Expected behavior

Expected to download the firecracker binary.

## Environment

`[Author TODO: Please supply the following information):]`
`[ - Firecracker version.]`
`[ - Host and guest kernel versions.]`
`[ - Rootfs used.]`
`[ - Architecture.]`
`[ - Any other relevant software versions.]`

## Additional context

`[Author TODO: How has this bug affected you?]`

`[Author TODO: What are you trying to achieve?]`

`[Author TODO: Do you have any idea of what the solution might be?]`
Updating the  [getting-started.md#getting-the-firecracker-binary](https://github.com/firecracker-microvm/firecracker/blob/master/docs/getting-started.md#getting-the-firecracker-binary) to new release flow.

## Checks
- [x] Have you searched the Firecracker Issues database for similar problems?
- [x] Have you read the existing relevant Firecracker documentation?
- [x] Are you certain the bug being reported is a Firecracker issue?

### PR DESCRIPTION
**Pull request by KarthikNedunchezhiyan**
## Reason for This PR

Closes #2394 

## Description of Changes

* Adjusted firecracker binary download flow to latest release model.

- [ ] This functionality can be added in [`rust-vmm`](https://github.com/rust-vmm).

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [x] All commits in this PR are signed (`git commit -s`).
- [ ] The reason for this PR is clearly provided (issue no. or explanation).
- [ ] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this PR.
- [ ] Any newly added `unsafe` code is properly documented.
- [ ] Any API changes are reflected in `firecracker/swagger.yaml`.
- [ ] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.

### CODE REVIEW

**Comment by raduiliescu** [comment]
Thanks for taking on this issue that escaped us. Change looks good to me.

**Review by raduiliescu — APPROVED** [review]

File: docs/getting-started.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ latest=$(basename $(curl -fsSLI -o /dev/null -w  %{url_effective} https://github
 ```
 
 ```wrap
-curl -LOJ https://github.com/firecracker-microvm/firecracker/releases/download/${latest}/firecracker-${latest}-$(uname -m)
+curl -L https://github.com/firecracker-microvm/firecracker/releases/download/${latest}/firecracker-${latest}-$(uname -m).tgz | tar -xz
```
Comment (dianpopa) [inline docs/getting-started.md:58]: I receive this error when trying to run this:
```
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

Why not
```
curl -LO https://github.com/firecracker-microvm/firecracker/releases/download/${latest}/firecracker-${latest}-$(uname -m).tgz
```
?

Comment (KarthikNedunchezhiyan) [reply docs/getting-started.md:58]: > I receive this error when trying to run this:
> 
> ```
> gzip: stdin: not in gzip format
> tar: Child returned status 1
> tar: Error is not recoverable: exiting now
> ```

I suspect $latest is not defined in your case so, github server may have responded with "Not Found" string and tar tried to parse it as a gzip format. can you check what is the output of `echo https://github.com/firecracker-microvm/firecracker/releases/download/${latest}/firecracker-${latest}-$(uname -m).tgz` 🙂

> Why not
> 
> ```
> curl -LO https://github.com/firecracker-microvm/firecracker/releases/download/${latest}/firecracker-${latest}-$(uname -m).tgz
> ```

Nothing much. Just tried to download and untar in a single line. 

If we use `-O` then the file will be saved to disk (with the name in the header) and then we have to untar the downloaded tar file. It takes two steps.

Comment (gc-plp) [reply docs/getting-started.md:58]: I think that this looks good. As @KarthikNedunchezhiyan pointed out, the `latest` variable needs to be defined first.

@dianpopa can you confirm that this works?

Comment (dianpopa) [reply docs/getting-started.md:58]: Indeed latest was not set. It works now.

**Review by dianpopa — APPROVED** [review]

**Review by gc-plp — APPROVED** [review]

**Pull request merged**